src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -150,15 +147,60 @@ export default class PhotoField extends React.Component {
 
   onZoom = (e) => {
     const newZoomValue = e.detail.ratio;
-    this.setState({ zoomValue: newZoomValue }, () => {
-      this.refs.slider.value = this.state.zoomValue;
-    });
+    if (newZoomValue < MAX_RATIO && newZoomValue > MIN_RATIO) {
+      let warningMessage;
+      if (this.state.zoomValue >= WARN_RATIO) {
+        warningMessage = 'Zooming in this close will make your ID picture less clear.';
```

Comment:
@peggygannon is this warning content alright?

Reply:
@cehsu Can you make this tweak to the copy:
If you zoom in this close, your ID photo will be less clear.

Reply:
Thanks @peggygannon, updated.